src/python/foglamp/data_purge/purge.py

```diff
@@ -162,23 +162,17 @@ def convert_timestamp(set_time: str) -> datetime.timedelta:
     return time_in_sec+time_in_min+time_in_hr+time_in_day
 
 
-def execute_command_with_return_value(stmt: str) -> dict:
+def execute_command(stmt: str):
     """Imitate connection to postgres that returns result.    
     Args:
         stmt (str): generated SQL query   
     Returns:
         Returns result set 
     """
-    query_result = _CONN.execute(stmt)
-    return query_result.fetchall()
-
-
-def execute_command_without_return_value(stmt: str) -> None:
-    """Imitate connection to Postgres and a query that doesn't generate results
-    Args:
-        stmt (str): DELETE stmt 
-    """
-    _CONN.execute(stmt)
+    engine = sqlalchemy.create_engine(__CONNECTION_STRING, pool_size=20, max_overflow=0)
+    conn = engine.connect()
+    query_result = conn.execute(stmt)
+    return query_result
 
 
 def set_id() -> int:
```

Comment:
this is not needed as foglamp.log id column is autoincrement:
`id    bigint NOT NULL DEFAULT nextval('foglamp.log_id_seq'::regclass),`